src/qibo/models/encodings.py

```diff
@@ -116,8 +116,69 @@ def phase_encoder(data, rotation: str = "RY", **kwargs):
     return circuit
 
 
+def binary_encoder(data, **kwargs):
```

Comment:
Could you share the reference/the idea you followed here to implement the binary encoding on this way?
Is that an alternative way to implement the "standard" binary encoding by means of controlled rotations?

Reply:
I'm not sure what you mean by "standard", since there are many ways to do binary encoding. This specific one is something that @GoGoKo699 and I have been working on, but it is not published yet. The only difference here is that the amplitudes are parameterized in a specific coordinate system in a $d$-dimensional hypersphere (in this case, called Hopf coordinates).